files/spring2020.md

```diff
@@ -9,6 +9,7 @@
 - [Dennis Murphy Playlist](https://www.youtube.com/watch?v=ET_9XHTvJEA&list=PLpTH_c-Pgrd_qfMSWBH5U6dx73w5bPsNg)
 - [AbstractFeel](https://www.youtube.com/playlist?list=PLJVIH4MDUvoGOIQcQSwdHeQkmu71YNUS_)
 - [btopro](https://btopro.com/)
+- {BLACKinTECH](https://www.youtube.com/channel/UCkjR9RZpsNaEFvbmYP2uZ4A/featured)
```

Comment:
in the future make sure it's `[thing]` got a the 1st character wrong there on the link name